README.md

```diff
@@ -6,13 +6,13 @@ The Windows Community Toolkit is a collection of helper functions, custom contro
 
 Do you want to contribute or update the docs? Read the [contribution guidelines](CONTRIBUTING.md)
 
-When opening a PR, start by forking this repository. Then, based on the type of change you're making you'll need to create a new branch from either the `master` or `live` branches:
+When opening a PR, start by forking this repository. Then, based on the type of change you're making you'll need to create a new branch from either the `master` or `dev` branches:
 
-:new::books: For documentation for new features, please base your fork off the master branch.
+:exclamation: If you have a typo or existing document improvement to an already shipped feature, please base your change off of the [Master branch](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/tree/master). This will allow us to get the change to the published documentation between releases.
```

Comment:
```suggestion
:exclamation: If you have a typo or existing document improvement to an already shipped feature, please base your change off of the [master branch](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/tree/master). This will allow us to get the change to the published documentation between releases.
```